selfdrive/loggerd/loggerd.cc

```diff
@@ -558,7 +558,7 @@ static void bootlog() {
     std::string lastPmsg = util::read_file("/sys/fs/pstore/pmsg-ramoops-0");
     boot.setLastPmsg(capnp::Data::Reader((const kj::byte*)lastPmsg.data(), lastPmsg.size()));
 
-    auto words = capnp::messageToFlatArray(msg);
+    auto words = msg.toFlatArrayPtr();
```

Comment:
I don't think we ever need the "words" ptr, maybe return after asBytes?

Reply:
just want to keep the parten with capnproto at that moment:return words arrary,let user choose howto convert it.

it looks unnecessary now.